### ISSUE TITLE
No Product Icon for Deskoo Menu Entries or shortcuts

### ISSUE THREAD

**Issue by michaelof** [a-sit/PDF-Over#65]
PDF-Over 4.4.4
OpenSUSE 15.4 (LINUX, Wayland)

Small issue. 

Icon have been there with 4.3.2-1. Disappeared with 4.4.0, AFAI remember. 
Still gone in 4.4.4

**Comment by iaik-jheher** [comment]
To clarify: the icon shows up on your desktop and works, but doesn't have the correct PDF-Over icon image set?

**Comment by michaelof** [comment]
Yes, a "PDF-Over" entry is made in Start Menu and on desktop. But it's a grey square and does not contain/show the nice product icon.
